### PR TITLE
Add cichy dataset.

### PR DESCRIPTION
**Pull request by jaeilepp**
For https://github.com/mne-tools/mne-python/pull/3552

### CODE REVIEW

**Comment by agramfort** [comment]
by looking at the data I realize there is 92 categories not 96...

also it's only 1 run.

@kingjr is it good enough to make a convincing example?

**Comment by kingjr** [comment]
> also it's only 1 run. @kingjr is it good enough to make a convincing example?

Well, from what I recall, the data for one subject isn't super clear; it's mainly face versus rest. But what alternative do we have? Many people ask me about RSA in MNE, it would be great to have an example..

**Comment by agramfort** [comment]
I agree but I am thinking that maybe packaging all the runs is the correct
thing to do even if it's "big data"...

**Comment by kingjr** [comment]
I'm for that, but I'm not sure we should package it as an example then.
it'd be closer to something alike MNE-HCP no?

**Comment by agramfort** [comment]
> I'm for that, but I'm not sure we should package it as an example then.
> it'd be closer to something alike MNE-HCP no?

I think we should package it but if it breaks the build of circle ci
we could make an example that does not start with plot_ so it's not
run when building the doc.

We need an RSA example in MNE.

An idea would be to look if we can make the data smaller eg package
downsampled data or maybe just the Epochs ...

any thought?

**Comment by jaeilepp** [comment]
Now all the runs and the trigger file are included.

File: mne/datasets/tests/test_datasets.py [hunk 1]
```diff
@@ -11,7 +11,8 @@ def test_datasets():
     """Test simple dataset functions
     """
     for dname in ('sample', 'somato', 'spm_face', 'testing',
-                  'bst_raw', 'bst_auditory', 'bst_resting'):
+                  'bst_raw', 'bst_auditory', 'bst_resting',
+                  'visual_96_categories'):
```
Comment (Eric89GXL) [inline mne/datasets/tests/test_datasets.py]: 92

**Comment by agramfort** [comment]
worked fine but I think that while you're at it you should add the T1.nii.gz and the visual_stimuli.mat file (maybe converted to a simpler .mat or .npy file of shape (92, n_rows, n_cols)

**Comment by jaeilepp** [comment]
> you should add the T1.nii.gz and the visual_stimuli.mat

Included. There's also the csv stimuli file.

**Comment by agramfort** [comment]
thx @jaeilepp

**Pull request merged**